apps/services/user-profile/project.json

```diff
@@ -119,7 +119,8 @@
         "parallel": true
       }
     },
-    "docker-express": {}
+    "docker-express": {},
+    "service-service-portal-api": {}
```

Comment:
Why is `service-service-portal-api` added?

Reply:
This is to connect back from service name - `service-portal-api` to the NX name that we then `yarn nx run ...`. So the convention is `service-<service-name>`. Would have be simpler if our NX names matched our service names, but that's too much work at this point I guess 🤷🏻‍♂️ .

Reply:
I was thinking we need to do it only for those names that do not match. But open to other ideas as well.

Reply:
I'm surprised `service-portal-api` works, since I've had issues in newer services if the infra service name doesn't match the NX name.